env/environment.yml

```
@@ -0,0 +1,26 @@
+name: isce3-ci-sandbox
```

Let's just name it `isce3`

```suggestion
name: isce3
```